lib/rspec/que/queue_up.rb

```diff
@@ -189,6 +206,72 @@ def failed_msg(candidates)
           end
         end
       end
+
+      class QueueCount
```

Comment:
👍  in principle but with the addition of a 70+-line matcher-helper-class I propose it's high time to put these classes into their own files.